workflow/python/ds-pack/v1/requirements.txt

```diff
@@ -0,0 +1,7 @@
+boto3==1.23.7
+pandas==1.4.2
+psycopg2==2.9.3
+snowflake==0.0.3
+snowflake-connector-python[pandas]==2.7.7
+sklearn==0.0
```

Comment:
Should this be version '0.0'? On a workspace I have I used '1.0.2'?

Reply:
Hey @vanessavirgo94. Is it okay if replace sklearn  with scikit-learn (1.1.1 - latest one) as sklearn pypi itself recommends using scikit-learn  (Ref- https://pypi.org/project/sklearn/)

Reply:
Yeah that is fine with me